r/README.md

```diff
@@ -178,6 +178,28 @@ For any other build/configuration challenges, see the [C++ developer
 guide](https://arrow.apache.org/docs/developers/cpp/building.html) and
 `vignette("install", package = "arrow")`.
 
+### Using `remotes::install_github(...)` 
+
+If you need an Arrow installation from a specific repository or at a specific ref, `remotes::install_github()` should work on most platforms. This method is helpful if you need a full install of arrow that is separate from another install (e.g. we use this in [arrowbench](https://github.com/ursacomputing/arrowbench) to install development versions of arrow isolated from the system install). However there are some caveats to be aware of:
+
+* Setting the environment variable `FORCE_TOOLS_LIBS_SCRIPT` to `true` will avoid linking to any arrow libraries already installed and attempt to build arrow from the same source at the repository+ref given.
+* If you are using the `FORCE_TOOLS_LIBS_SCRIPT` you must also set `build = FALSE` in the `remotes::install_github()` call. This is similar to checking out the repository and calling `R CMD INSTALL .` in the `arrow/r` directory (as opposed to first calling `R CMD BUILD .` and then installing the tar.gz file that produces, which is the default for `remotes::install_github()`).
+* Specify `subdir = "r"` to get the R package.
```

Comment:
I believe it is preferred now that you specify the subdir as part of `repo` now, i.e. `remotes::install_github("apache/arrow/r", build = FALSE)`  cf. https://remotes.r-lib.org/reference/install_github.html

Reply:
TIL, lemme double check that this does the same thing (I can't imagine it doesn't, but just in case!